docs/guides/security/backups/create-restic-repository-shortguide/index.md

```diff
@@ -41,31 +45,43 @@ the repository. Losing your password means that your data is
 irrecoverably lost.
 {{< /output >}}
 
-{{< caution >}}
+    {{< caution >}}
 Store this password securely and somewhere away from your Linode. Your backups will be inaccessible without it!
 {{< /caution >}}
 
 ### Store the access key and secret
 
 Your access key, secret key and password are required every time Restic communicates with your repository. To make it easier to work with your repository, create a shell script containing your credentials.
 
-To keep it secure, create this script within the root user's home directory, and run all your Restic scripts as the root user.
+1. To keep it secure, create this script within the root user's home directory, and run all your Restic scripts as the root user.
 
-    sudo nano /root/restic_params
+        sudo nano /root/restic_params
 
-{{< file "/root/restic_params" >}}export AWS_ACCESS_KEY_ID=your-key
+    {{< file "/root/restic_params" >}}
+export AWS_ACCESS_KEY_ID=your-key
 export AWS_SECRET_ACCESS_KEY=your-secret
 {{< /file >}}
 
+    {{< note >}}
 Whenever you want to use Restic, import this file or include it in your user's logon script:
 
     source /root/restic_params
+{{< /note >}}
+
+1. Create a password file to hold your Restic password:
 
-Create a password file to hold your Restic password:
+        sudo nano /root/restic_pw
```

Comment:
The previous step had this command, so I thought I would put it here too for consistency